CvGameCoreDLL_Expansion2/CvReligionClasses.cpp

```diff
@@ -5776,8 +5776,10 @@ void CvCityReligions::RecomputeFollowers(CvReligiousFollowChangeReason eReason,
 		}
 
 		if (iLargestRemainder > 0)
+		{
```

Comment:
TBH, at this point we should have run out of unassigned followers and existed the for-loop, so the if statement itself is probably unnecessary. Better safe than sorry!